pnpm-workspace.yaml

```diff
@@ -19,3 +18,9 @@ catalog:
   glob: "^10.3.10"
   gray-matter: "^4.0.3"
   "@types/glob": "^8.1.0"
+
+catalogs:
+  examples:
+    "@types/node": "^22"
```

Comment:
Why v22 for examples?

Reply:
If this is necessary, we should document it somewhere.

Reply:
I chose 22 as it is the current active version.